lib/config.js

```diff
@@ -28,10 +27,12 @@ const defaults = {
 };
 
 function loadConfigs(path) {
-  const configPath = path || Path.resolve(__dirname, '../config/dev.json');
-  const configs = nconf.file(configPath);
+  let configs = nconf.defaults(defaults);
```

Comment:
This is a weird usage of nconf. Generally, configuration is a singleton, so the nconf package actually exports an instantiated config object, which you attach sources to:

```javascript

var Config = globals.Confog = require('nconf');
Config.argv(...);
Config.file(...);
Config.defaults({
...
});

module.exports = Config;
```